src/js/components/node/profile.jsx

```diff
@@ -258,8 +264,8 @@ let ProfileNode = React.createClass({
     }
   },
 
-  _handleCopyURL() {
-
+  _handlePostCopyURL() {
+    alert('The URL was copied to the clipboard.')
```

Comment:
Can you add a  @TODO comment noting that we need to implement snackbars/toasts (https://material.google.com/components/snackbars-toasts.html#snackbars-toasts-usage)